### ISSUE TITLE
Chapter 4: bgd_path

### ISSUE THREAD

**Issue by ChensStudio** [ageron/handson-ml2#68]
in Chapter 4 bgd_path function, why is it divided by 2 for l1 value (N/2) in penalty contour plots?
ax.contourf(t1, t2, N / 2., levels=levelsN)

Thank you for writing this awesome book.

**Comment by ageron** [comment]
Thanks for your kind words @ChensStudio! If you feel like writing an [Amazon review](https://www.amazon.com/Hands-Machine-Learning-Scikit-Learn-TensorFlow/dp/1492032646/ref=dp_ob_title_bk), that would be much appreciated! :)

Regarding N/2, I believe I did that to make the graph look a bit nicer. If I remember correctly, using N instead led to ugly color gradients (you can give it a try if you want). In short, it's just a formatting detail, nothing important at all.

I hope you'll enjoy the rest of the book! :))
